workers/ohsome_quality_analyst/oqt.py

```diff
@@ -67,8 +71,9 @@ async def _(
         logging.info("Input feature identifier: " + id_)
         if size_restriction:
             await check_area_size(feature.geometry)
-        indicator = await create_indicator(parameters.copy(update={"bpolys": feature}))
-        features.append(indicator.as_feature(flatten=True))
+        tasks.append(create_indicator(parameters.copy(update={"bpolys": feature})))
+    indicators = await asyncio.gather(*(sem_task(task) for task in tasks))
```

Comment:
Didn't try it, but wouldn't it be easier to apply sem_task directly?
```suggestion
        tasks.append(sem_task(create_indicator(parameters.copy(update={"bpolys": feature}))))
    indicators = await asyncio.gather(*tasks)
```